### PR TITLE
Add CHANGELOG

### PR DESCRIPTION
**Pull request by onimsha**
Signed-off-by: Alex <alex.tuan@mindvalley.com>

## Summary

Ticket:

## What's Changed

* Add CHANGELOG.md

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/48#" title="Author self-approved">onimsha</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: README.md
Comment (jk-gan) [inline README.md:60]:
```suggestion
## IMPORTANT !!

The latest release of Wukong CLI contains several breaking changes. Users must upgrade to the latest version, which is at least `0.0.4-beta1` in order to continue using the Wukong CLI. If you are using Wukong version < `0.0.4-beta1`, please follow the below instructions:

```
# Check the current version of Wukong CLI.  
$ wukong --version

# If your version is < 0.0.4-beta1.  
$ brew update  
$ brew upgrade wukong  

# Confirm that you're on version >= 0.0.4-beta1.  
$ wukong --version
wukong 0.0.4-beta1

# Delete the current config.  
$ rm ~/.config/wukong/config.toml

# Re-init the Wukong CLI.  
$ wukong init
```

```

**Review by jk-gan — COMMENTED** [review]
I think the generated changelog is pretty good!

File: README.md
Comment (jk-gan) [inline README.md:19]: it is better to show the logo on the top, and still show the cli help screenshot here

**Review by jk-gan — CHANGES_REQUESTED** [review]

**Review by jk-gan — APPROVED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/48#pullrequestreview-1212954219" title="Approved">jk-gan</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/48#" title="Author self-approved">onimsha</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
